test/parallel/taskCompare/elliot/empty-omp-parallel-for.compopts

```diff
@@ -0,0 +1 @@
+ompTaskSpawn.h
```

Comment:
Intentionally left that off because I didn't want to have to worry about fopenmp portability to various compilers (which is also why I guarded everything omp in the .c file with _OPENMP.)